### PR TITLE
Fix#1282 Add VaSelect 'create-new' emit

### PR DESCRIPTION
**Pull request by RVitaly1978**
## Description
close: #1282

Changes:
- [x] - add `create-new` emit
- [x] - add translations
- [x] - update doc and demo examples

chrome-capture

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Improvement/refactoring (non-breaking change that doesn't add any feature but make things better)

### CODE REVIEW

File: packages/docs/src/locales/en/en.json [hunk 1]
```diff
@@ -758,7 +758,8 @@
       },
```
Comment (asvae) [inline packages/docs/src/locales/en/en.json:758]: On mouse down:

image

Normal:

image

You can see outline on click. That shouldn't happen.

File: packages/docs/src/locales/en/en.json [hunk 2]
```diff
@@ -758,7 +758,8 @@
...
       "events": {
```
Comment (asvae) [inline packages/docs/src/locales/en/en.json:759]: When i focus select and i press tab - I expect next select to be focused.
Instead focus is lost.

File: packages/docs/src/locales/en/en.json [hunk 3]
```diff
@@ -758,7 +758,8 @@
...
         "clear": "Emitted if select value has been cleared",
```
Comment (asvae) [inline packages/docs/src/locales/en/en.json:760]: Pressing Enter should clear search text when allow-create flag is true.

image

File: packages/docs/src/locales/en/en.json [hunk 4]
```diff
@@ -758,7 +758,8 @@
...
-        "updateSearch": "Emitted if search value has changed"
```
Comment (asvae) [inline packages/docs/src/locales/en/en.json:761]: When I press UP button - cursor goes directly to "five" option.
It should not ignore the option in-between even though text value is the same (id is not!)

image

**Comment by asvae** [comment]
Found a couple of things - let's see if we can fix these quickly. If no - let's move fixing these to another PR (apart from shadow things are not critical)

File: packages/docs/src/locales/en/en.json (hunk 1 above)
Comment (RVitaly1978) [reply packages/docs/src/locales/en/en.json:758]: Fixed it

File: packages/docs/src/locales/en/en.json (hunk 3 above)
Comment (RVitaly1978) [reply packages/docs/src/locales/en/en.json:760]: Fixed it

File: packages/docs/src/locales/en/en.json (hunk 4 above)
Comment (RVitaly1978) [reply packages/docs/src/locales/en/en.json:761]: Let's move fixing these to another PR

File: packages/docs/src/locales/en/en.json (hunk 2 above)
Comment (RVitaly1978) [reply packages/docs/src/locales/en/en.json:759]: I have this behavior when press `tab`
chrome-capture

File: packages/docs/src/locales/en/en.json (hunk 4 above)
Comment (RVitaly1978) [reply packages/docs/src/locales/en/en.json:761]: fixed it:
chrome-capture

**Comment by asvae** [comment]
Thanks 🤗

**Pull request merged**